### PR TITLE
Chore: SDLC Writeup Link

### LINKED ISSUE #13: Link the SDLC Writeup on the Front-Page of the Project
Add a section that tackles OWASP in SDLC and links to the write-up accordingly

### PR DESCRIPTION
**Pull request by ThunderSon**
This PR takes care of:
- Update the URL name
- Set a text on the front page to link to the SDLC Writeup

### CODE REVIEW

**Comment by ThunderSon** [comment]
@kingthorin your approval to merge this,

File: _config.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 remote_theme: "owasp/www--site-theme"
 plugins:
- - jekyll-include-cache-0.2.0
\ No newline at end of file
+ - jekyll-include-cache-0.2.0
+  - jekyll-redirect-from
```
Comment (kingthorin) [inline _config.yml]:
```suggestion
 - jekyll-include-cache-0.2.0
 - jekyll-redirect-from
 
```

File: index.md [hunk 2]
```diff
@@ -24,3 +24,7 @@ The below is a rough estimation as it is dependent on other projects' time and a
 - End of Q3 2020: MVP release where a proper requirements set has been established and released for testing to be conducted on the standard being proposed.
 
 The roadmap will be adjusted as the project moves forward.
+
+## OWASP In the SDLC
```
Comment (kingthorin) [inline index.md]:
```suggestion
## OWASP in the SDLC
```

File: writeups/owasp_in_sdlc/index.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: OWASP in SDLC
 layout: col-document
 tags: SDLC standards integration
 document: integration standards
+redirection_from: "/writeups/owasp_in_sdlc/owasp_in_sdlc.html"
```
Comment (kingthorin) [inline writeups/owasp_in_sdlc/index.md]: Do you know that it's linked elsewhere as *.html?
You may also want to redirect from: https://owasp.org/www-project-integration-standards/writeups/owasp_in_sdlc/owasp_in_sdlc

(People shouldn't be using filename extensions.)

**Review by kingthorin — CHANGES_REQUESTED** [review]

File: writeups/owasp_in_sdlc/index.md (hunk 3 above)
Comment (ThunderSon) [reply writeups/owasp_in_sdlc/index.md]: Nope, didn't know that. Will remove the .html

File: _config.yml (hunk 1 above)
Comment (ThunderSon) [reply _config.yml]: One space? That's weird. Shouldn't they both be 2 spaces instead?

Comment (kingthorin) [reply _config.yml]: I was just going for consistency and the file ending with a blank line 👍

**Review by kingthorin — APPROVED** [review]

**Pull request merged**